### PR TITLE
Tune remediation points

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
Tune complexity points, following calculation function in [codeclimate-eslint](https://github.com/codeclimate/codeclimate-eslint/blob/master/lib/checks.js#L117), creating more parity with Ruby analysis in Code Climate classic.

cc @codeclimate/review @noahd1

### CODE REVIEW

File: spec/cc/engine/rubocop_spec.rb [hunk 1]
```diff
@@ -323,7 +323,7 @@ def includes_check?(output, cop_name)
       def includes_content_for?(output, cop_name)
         issue = issues(output).detect { |i| i["check_name"] =~ /#{cop_name}$/ }
 
-        issue["content"]["body"].present?
+        issue["content"] && issue["content"]["body"].present?
```
Comment (ABaldwinHunter) [inline spec/cc/engine/rubocop_spec.rb]: This change isn't related, but came up when running tests. Happy to remove for cleaner diff if preferred.

Comment (pbrisbin) [inline spec/cc/engine/rubocop_spec.rb]: I'm fine with this drive-by, but I'd make it its own commit if you can.

File: config/cops.yml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-  base_points: 30_000
+  base_points: 100_000
 Metrics/ClassLength:
-  base_points: 400_000
+  base_points: 800_000
+  violation_points: 75_000
 Metrics/CyclomaticComplexity:
-  base_points: 75_000
-  violation_points: 10_000
+  base_points: 1_000_000
```
Comment (mrb) [inline config/cops.yml]: Does this effectively mean that any instance of Cyclomatic complexity makes a file a "B"?

File: spec/cc/engine/rubocop_spec.rb (hunk 1 above)
Comment (ABaldwinHunter) [inline spec/cc/engine/rubocop_spec.rb]: Noted. Moved to separate commit.

File: config/cops.yml (hunk 2 above)
Comment (ABaldwinHunter) [inline config/cops.yml]: 2,000,000 remediation points are needed in penalties to go from A to B. 

So not always, but a lot of the time an instance of Cyclomatic Complexity could have that large of grade impact.

I was talking with @brynary yesterday and had the impression that that level of grade impact would be _close_ to what we want for complexity - but maybe requires some additional discussion?

**Comment by ABaldwinHunter** [comment]
@codeclimate/review @brynary 

Ready for another look!

**Comment by ABaldwinHunter** [comment]
from commit message:
```
In Classic Analysis, typical remediation point formular for complexity were
roughly:

Class
  5_000_000 + 50_000n

Method
  1_000_000 + 100_000n

This change updates codeclimate-rubocop to be closer to those standards.

Code Climate users can turn on one or all complexity checks, with the
understanding that there may be overlap between them.

Users can also configure thresholds in their .rubocop.yml file.
```

**Comment by ABaldwinHunter** [comment]
Ready for re-review.

**Comment by ABaldwinHunter** [comment]
cc @codeclimate/review

File: spec/cc/engine/rubocop_spec.rb (hunk 1 above)
Comment (GordonDiggs) [inline spec/cc/engine/rubocop_spec.rb]: I still only see one commit on this branch

Comment (ABaldwinHunter) [inline spec/cc/engine/rubocop_spec.rb]: Ahhh sorry. 
Fixed. Then made additional changed and squashed everything again. My bad. Updating.

Comment (ABaldwinHunter) [inline spec/cc/engine/rubocop_spec.rb]: Done! Thanks for the catch.

**Comment by GordonDiggs** [comment]
LGTM

File: config/cops.yml [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
+  violation_points: 35_000
+Metrics/CyclomaticComplexity: # This check is per method
+  base_points: 1_000_000
+  violation_points: 70_000
 Metrics/LineLength:
-  base_points: 20_000
-  violation_points: 5_000
-Metrics/MethodLength:
-  base_points: 50_000
+  base_points: 100_000
```
Comment (brynary) [inline config/cops.yml]: Is `LineLength` for a single LOC being too long (wide)? If so, this seems too high.

**Comment by brynary** [comment]
Reviewed again. Had one question about `LineLength` (added inline). Everything else LGTM

File: config/cops.yml (hunk 3 above)
Comment (ABaldwinHunter) [inline config/cops.yml]: Yes, concur that's the offense.

Reverting to the [current setting](https://github.com/codeclimate/codeclimate-rubocop/blob/master/config/cops.yml#L10):
```
Metrics/LineLength:
  base_points: 20_000
  violation_points: 5_000
```

if that sounds good?

Comment (brynary) [inline config/cops.yml]: So for a style thing (like missing semicolon in JS) we do 50k. I think that applies here as well. I'm not sure anything would ever be less than 50k, since the 50k number is intended to account for the time it takes to open the file, make the change, save, commit, push... it's about 5 mins even for something small.

For a long line, it doesn't seem to get harder to fix as the line gets longer. Therefore, I'd just do a flat 50k.
